iceoryx_examples/waitset/README.md

```diff
@@ -133,16 +133,17 @@ A _Trigger_ always requires a callback which has the following signature
 has received (`take()`). When `take()` was successful we print our message to
 the console inside of the `and_then` lambda.
 
-In our `main` function we create a _WaitSet_ after we registered at our central
-broker RouDi and attach our `shutdownTrigger` to it to handle `CTRL+c` events. 
+In our `main` function we create a _WaitSet_ which can hold a capacity of 5 triggers
+(4 subscribers and the shutdown trigger) after we registered us at our central
+broker RouDi. Then we attach our `shutdownTrigger` to it to handle `CTRL+c` events. 
 ```cpp
-iox::popo::WaitSet waitset;
+iox::popo::WaitSet waitset<NUMBER_OF_SUBSCRIBERS + 1>;
```

Comment:
the `1` changed to `ONE_SHUTDOWN_TRIGGER`

Reply:
done